bytelatent/base_transformer.py

```
@@ -16,7 +16,10 @@
 
 from bytelatent import probe
 
-flex_attention_comp = torch.compile(flex_attention)
+try:
```

For both of these, could you gate the changed behavior around a variable like `BLT_ALLOW_MISSING_FLEX_ATTENTION`? E.G., something like this:
```python
if 'BLT_ALLOW_MISSING_FLEX_ATTENTION' in os.environ and int(os.environ['BLT_ALLOW_MISSING_FLEX_ATTENTION']) == 1:
    # do custom behavior
else:
    # default to current behavior
```
The main reason for this is that if for some reason flex attention is not importing correctly yet it would run on the hardware, we want our scripts to fail rather than silently run slowly. This allows that to happen while still providing an opt out mechanism.

Thanks for PR, let me know when you want me to take a second look.

---

Just added the environment variable you requested. Let me know if you need anything else!